### ISSUE TITLE
fn_call_style config makes rustfmt add extra comma for single-line function

### ISSUE THREAD

**Issue by daboross** [rust-lang-nursery/rustfmt#1501]
With the config:

```toml
max_width = 120
fn_call_style = "block"
```

this function is reformatted:
```diff
 fn main() {⏎
-    let hyper = Arc::new(Client::with_connector(HttpsConnector::new(hyper_rustls::TlsClient::new())));⏎
+    let hyper = Arc::new(Client::with_connector(HttpsConnector::new(hyper_rustls::TlsClient::new())),);⏎
 }⏎
```

Not sure how widespread this is, just thought I'd report it here. Hope this helps!

**Comment by nrc** [comment]
Have you tried this with master? A number of similar bugs have been fixed recently.

**Comment by daboross** [comment]
Yes, this is with a freshly pulled clone, c89c01efc0b14f5dff93088243d833b30a9e3ad4

**Comment by topecongiro** [comment]
Hi, I created a PR (#1442) a month ago related to this issue, and forgot to update it. I would appreciate if you could try it.

**Comment by daboross** [comment]
@topecongiro Indeed, that PR does fix this issue!

**Comment by topecongiro** [comment]
@daboross Thanks!

**Comment by topecongiro** [comment]
Closed by #1442.
